src/oralb_ble/parser.py

```diff
@@ -224,6 +231,14 @@ class ModelDescription:
 class OralBBluetoothDeviceData(BluetoothData):
     """Data for OralB BLE sensors."""
 
+    def __init__(self):
+        super().__init__()
+        # If this is True, then we have not seen an advertisement with a payload
+        self.pending = True
```

Comment:
Let's name it to better show intent

```suggestion
        self._seen_advertisement = False
```

Reply:
Changing. I originally made it public as Xiaomi sometimes calls .pending in the ha component, but I think since we don't deal with the same issue of not knowing the device/ encryption, we should be good to just keep it local.